### PR TITLE
Ab#49294 include the option for user to identify if they are applying for a certificate renewal

### PR DESCRIPTION
**Pull request by AlexandreSanscartier**

### CODE REVIEW

File: CDTS Blazor/Models/PageModels/UploadDocumentPageModel.cs [hunk 1]
```diff
@@ -2,14 +2,21 @@
...
 
     /// <summary>
     /// PageModel for the UploadDocument page.
     /// </summary>
     public class UploadDocumentPageModel
     {
+        /// <summary>
+        /// Gets or sets renewal status.
+        /// </summary>
+        public bool IsRenewal { get; set; } = false;
```
Comment (TCPrimedPaul) [inline CDTS Blazor/Models/PageModels/UploadDocumentPageModel.cs:18]: I don't see any references to this member in code, is it needed?

Comment (AlexandreSanscartier) [reply CDTS Blazor/Models/PageModels/UploadDocumentPageModel.cs:18]: Good point going to remove it thanks!

File: CDTS Blazor/Pages/UploadDocument.razor [hunk 2]
```diff
@@ -90,6 +92,52 @@
...
+                <input type="radio" name="animal" id="animal-2">
+                <label for="animal-2">Cat</label>
+            </li>
+            <li class="radio">
+                <input type="radio" name="animal" id="animal-3">
+                <label for="animal-3">Rat</label>
+            </li>
+        </ul>
+    </fieldset>
+                            -->
```
Comment (TCPrimedPaul) [inline CDTS Blazor/Pages/UploadDocument.razor:135]: I don't see the option for parakeets :clown_face:
I hear it's not good practice to leave unused code, maybe your forgot about this.

**Comment by TCPrimedPaul** [comment]
Great work!

**Review by TCPrimedPaul — APPROVED** [review]
Great work!!

**Pull request merged**